studio/pages/docs/tables/[tableName].js

```diff
@@ -0,0 +1,88 @@
+import DocsLayout from '../../../components/layouts/DocsLayout'
+import Loading from '../../../components/utils/Loading'
+import Error from '../../../components/utils/Error'
+import { fetchOpenApiSpec } from '../../../lib/api'
+import { useRouter } from 'next/router'
+import { Typography } from '@supabase/ui'
+
+export default function Home() {
```

Comment:
Curious as to why you've went with `.js` for the `auth/` and `docs/` folders?

Reply:
💯 With you 100% on this, I just switched all these components to `.tsx` and already prevented a few potential runtime bugs popping up.

Reply:
fixed by @i-pip ⚡